### ISSUE TITLE
Exception calling native with command :: CordovaStripe :: setPublishableKey ::exception=TypeError: errorCallback is not a function

### ISSUE THREAD

**Issue by gujiman** [zyramedia/cordova-plugin-stripe#11]
hi,

i created a blank V2 app with the latest stripe plugin. i can get the android version to work. However i'm having problem with the browser version. I get the following error:

```
Exception calling native with command :: CordovaStripe :: setPublishableKey ::exception=TypeError: errorCallback is not a function

Exception calling native with command :: CordovaStripe :: createCardToken ::exception=TypeError: errorCallback is not a function
```

------------------------------------------------------------------------------
What i'm using:
    "@angular/common": "2.4.8",
    "@angular/compiler": "2.4.8",
    "@angular/compiler-cli": "2.4.8",
    "@angular/core": "2.4.8",
    "@angular/forms": "2.4.8",
    "@angular/http": "2.4.8",
    "@angular/platform-browser": "2.4.8",
    "@angular/platform-browser-dynamic": "2.4.8",
    "@angular/platform-server": "2.4.8",
    "@ionic/storage": "2.0.0",
    "ionic-angular": "2.2.0",
    "ionic-native": "2.4.1",
    "ionicons": "3.0.0",
    "rxjs": "5.0.1",
    "sw-toolbox": "3.4.0",
    "zone.js": "0.7.2"

**Comment by ihadeed** [comment]
@gujiman are you using the Ionic Native wrapper? or using the plugin directly?

**Comment by gujiman** [comment]
Hi @ihadeed im using the ionic Native wrapper, with the sample code provided in the docs.

**Comment by ihadeed** [comment]
Fixed and now available in latest version `1.4.0`.
